dev-python/dogpile-cache/dogpile-cache-0.6.4.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=5
```

Comment:
Ye, didn't see that it was still on EAPI 5. I wish repoman would throw a warning on older EAPIs.